### PR TITLE
Add Redis Cloud free t-shirt

### LINKED ISSUE #533: RedisLabs free t-shirt
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : RedisLabs

Q. Where have you seen the announcement (URL)?  
A. : https://redislabs.com/try-redis-cloud-essentials-for-a-free-shirt/

Q. What kind of swag (clothing, stickers, ...)?  
A. : t-shirt

Q. Have you received swag from this?  
A. : No

Q. Does it ship worldwide?  
A. : No, only US and Canada

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by brittbrouwer**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added a free t-shirt from Redis Cloud while signing up for the trial

Fixes #533

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -565,5 +565,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
-  }
+  },
+  {
+    "name": "Redis Cloud",
```
Comment (aslafy-z) [inline data.json]: Please reorder this entry by name.

File: data.json [hunk 2]
```diff
@@ -565,5 +565,14 @@
...
+    "difficulty": "easy",
+    "description": "Sign up for an account. Once you create your first database, you will receive a free t-shirt.",
+    "reference": "https://redislabs.com/try-redis-cloud-essentials-for-a-free-shirt/",
+    "image": "https://redislabs.com/wp-content/uploads/2019/12/Redislabs-Cube-T-shirt-01.png",
+    "dateAdded": "2020-02-20T21:00:00.000Z",
```
Comment (aslafy-z) [inline data.json]: Please update this date with the opening date of the associated issue.

**Comment by aslafy-z** [comment]
Please add `Fixes #id-of-initial-issue` into your original PR body. Thanks!

**Comment by lucasqueiroz** [comment]
Would be cool if it mentioned it's North America only

File: data.json [hunk 3]
```diff
@@ -565,5 +565,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
-  }
+  },
+  {
+    "name": "Redis Cloud",
+    "difficulty": "easy",
+    "description": "Sign up for an account. Once you create your first database, you will receive a free t-shirt.",
```
Comment (aslafy-z) [inline data.json:572]:
```suggestion
    "description": "Sign up and create your first database, you will receive a free t-shirt. (Ships only to North America)",
```

File: data.json (hunk 2 above)
Comment (aslafy-z) [reply data.json:575]:
```suggestion
    "dateAdded": "2020-02-14T17:13:48.000Z",
```

**Comment by aslafy-z** [comment]
I went ahead and made the requested changes. Thanks for your contribution @brittbrouwer!

**Pull request merged**